stack/indexer/deb/debian/postinst

```diff
@@ -14,12 +14,13 @@ export USER=${NAME}
 export GROUP=${NAME}
 export CONFIG_DIR=/etc/${NAME}
 export INSTALLATION_DIR=/usr/share/${NAME}
+export BACKUP_DIR=/usr/share/backup
```

Comment:
If possible create the backup inside the installation directory

Reply:
Done. Tested in the forked repository:
- https://github.com/davidcr01/wazuh-packages/actions/runs/3873129841
- https://github.com/davidcr01/wazuh-packages/actions/runs/3873129836